src/TestProblem/ELBDM/LSS/Init_TestProb_ELBDM_LSS.cpp

```diff
@@ -234,6 +346,24 @@ void Init_ByFile_ELBDM_LSS( real fluid_out[], const real fluid_in[], const int n
 #endif // #if ( MODEL == ELBDM )
 
 
+//-------------------------------------------------------------------------------------------------------
+// Function    :  End_Region_LSS
+// Description :  Free memory before terminating the program
+//
+// Note        :  1. Linked to the function pointer "End_User_Ptr" to replace "End_User()"
+//
+// Parameter   :  None
+//-------------------------------------------------------------------------------------------------------
+void End_Region_LSS()
+{
+  delete [] ZoomIn_a;
+  delete [] ZoomIn_Lx;
+  delete [] ZoomIn_Ly;
+  delete [] ZoomIn_Lz;
+  delete [] ZoomIn_Center_x;
+  delete [] ZoomIn_Center_y;
+  delete [] ZoomIn_Center_z;
+}
```

Comment:
[Style] There are three indent spaces and add a comment at the end of the function parenthesis:
```suggestion
void End_Region_LSS()
{
   delete [] ZoomIn_a;
   delete [] ZoomIn_Lx;
   delete [] ZoomIn_Ly;
   delete [] ZoomIn_Lz;
   delete [] ZoomIn_Center_x;
   delete [] ZoomIn_Center_y;
   delete [] ZoomIn_Center_z;
} // FUNCTION : End_Region_LSS
```